jupyterhub/tests/selenium/test_browser.py

```diff
@@ -209,36 +225,686 @@ async def test_open_url_login(
         ("user", "password"),
     ],
 )
-async def test_invalid_credantials(app, browser, user, pass_w):
-    await open_url(app, browser)
+async def test_login_with_invalid_credantials(app, browser, user, pass_w):
+    await open_url(app, browser, param="login")
     await login(browser, user, pass_w)
-    await asyncio.sleep(0.1)
+    await asyncio.sleep(0.5)
```

Comment:
Let's try to add a comment explaining what we are waiting for with each sleep, so we can maybe replace them with specific conditions in the future.